Runtime/Util/RequestsUtil.cs

```diff
@@ -77,15 +77,15 @@ private static IEnumerator DoRequest(IAsyncCompletionSource<UnityWebRequest> op,
             }
 
             // Sometimes the webrequest is finished but the download is not
-            while (!webRequest.isNetworkError && !webRequest.isHttpError && webRequest.downloadProgress != 1)
+            while (webRequest.result != UnityWebRequest.Result.ConnectionError && webRequest.result != UnityWebRequest.Result.ProtocolError && webRequest.downloadProgress != 1)
```

Comment:
This should be done using the #if UNITY_2022_OR_NEWER and have an else with the old code

Reply:
fixed